projects/canopy/src/lib/skeleton/skeleton.directive.ts

```diff
@@ -0,0 +1,62 @@
+import { Directive, ElementRef, Input, HostBinding, OnDestroy } from '@angular/core';
+
+@Directive({
+  selector: '[lgSkeleton]',
+})
+export class LgSkeletonDirective implements OnDestroy {
+  private changes: MutationObserver;
+  private hasContent = false;
+
+  @Input() lgSkeletonWidth: string = null;
```

Comment:
Could you not leave the default values unspecified as they are set to be falsey anyway?